### ISSUE TITLE
Is there a way to exclude templates from offline compression ?

### ISSUE THREAD

**Issue by spapas** [django-compressor/django-compressor#1022]
I'm using django-nested-inlines that has a bunch of templates that are related to django-grapelli but I don't use grapelli (thus these templates don't get accessed). Now when I try to run manage.py compress I get:

```
source /home/serafeim/edla/venv/bin/activate&&python manage.py compress
Compressing... Invalid template /home/serafeim/edla/venv/lib64/python3.6/site-packages/nested_admin/templates/nesting/admin/includes/grappelli_inline.html: 'grp_tags' is not a registered tag library. Must be one of:
...
```

followed by a list of my installed packages. Among them is the tag libraries of my installed apps (and of course django grapelli ain't among them). So, ideally I'd like to exclude that template completely from offline compression. Is that possible somehow ?

**Comment by spapas** [comment]
Followin this and after taking a quick peek at the compress management command I presume it would be rather easy to add a multiple `-x// --exclude` command line option for excluding specific templates from offline compression. Would it be ok if I implemented that functionality?

**Comment by jorrit-wehelp** [comment]
Also see https://github.com/django-compressor/django-compressor/issues/785
(Sounds a bit easier to glob/regexp filter on the file path compared to working with app_labels like --exclude does for dumpdata)

**Comment by spapas** [comment]
Hello @jorrit-wehelp yes the issue you mention is exactly what I wanted! Is somebody working on that feature? I'd be happy to provide a PR if nobody else wants to implement it.

**Comment by karyon** [comment]
thanks @jorrit-wehelp for digging that out!

@spapas I don't think someone's working on it, you're welcome to do so!

Closing as duplicate in favor of #785.

**Comment by spapas** [comment]
I've submitted the PR #1024 that should fix #785 (and of course this duplicate issue).

BR,
Serafeim
